src/core/klsh.js

```
@@ -0,0 +1,106 @@
+      args.push(argObj.text);
+    }
+    const builtin = klsh[name];
+    if (builtin && typeof builtin.main === 'function') {
+      try {
+        const res = builtin.main({ args, stdin: '', env });
+        stdout += res.stdout;
+        stderr += res.stderr;
+        env = clone(res.env);
+      } catch (err) {
+        // Report only the error message; optionally include full error if verbose flag is present
+        stderr += (err.message || 'Error') + '\n';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)